node/env0-deploy-flow.js

```diff
@@ -25,7 +25,10 @@ const createAndDeploy = async (options, environmentVariables) => {
   await setConfigurationFromOptions(environmentVariables, environment, options.blueprintId);
   await deployUtils.pollEnvironmentStatus(environment.id);
   await deployUtils.deployEnvironment(environment, options.revision, options.blueprintId);
-  await deployUtils.pollEnvironmentStatus(environment.id);
+  const lastStatus = await deployUtils.pollEnvironmentStatus(environment.id);
+  if (lastStatus != 'ACTIVE') {
```

Comment:
should it fail on waiting for approval status?

Reply:
@onhate - great catch, it shouldn't fail on that.

Reply:
https://github.com/env0/env0-client-integrations/issues/8